CONTRIBUTING.md

```diff
@@ -124,7 +124,7 @@ $ TEST_DEBUG=true make test
 To test the code coverage, use:
 
 ```sh
-$ make test-cov
```

Comment:
Ah that makes sense. I'll close this and just create an issue pointing out the inconsistency between CONTRIBUTING.md and the current Makefile.